src/App.js

```diff
@@ -1,10 +1,12 @@
 import React from "react";
+import PokeCard from "./components/PokeCard";
 import "./styles.css";
 
 export default function App() {
   return (
     <div className="App">
       <h1>Welcome to your second mini challenge!</h1>
+      <PokeCard />
```

Comment:
It is better if you send mock data here as properties instead of loading everything inside the component. In that way, you can reuse the component at other contexts